sdks/java/core/src/test/java/org/apache/beam/sdk/util/ApiSurfaceTest.java

```diff
@@ -76,7 +68,33 @@ public void testOurApiSurface() throws Exception {
   }
 
   private boolean classIsAllowed(Class<?> clazz) {
-    return  clazz.getName().startsWith("org.apache.beam");
+    return clazz.getName().startsWith("org.apache.beam")
```

Comment:
I'd suggest making this a set of either prefixes or `Predicate<String>`s. That will make it easier to add more entries than this. Then this can just be:

```
for (String prefix : ALLOWED_PREFIXES) {
  if (clazz.getName().startsWith(prefix)) {
    return true;
  }
}
return false;
```

As a separate question, how do we deal with cases where we want to allow some parts of a package but not all of it. Eg., the testing sub-package may be different from main package.